Tools/autotest/sim_vehicle.py

```diff
@@ -191,7 +191,7 @@ def under_macos():
 
 
 def under_vagrant():
-    return os.path.isdir("/vagrant")
+    return os.path.isfile("/ardupilot.vagrant")
```

Comment:
You're reverting a commit already on master in here.

A couple, actually.

We have a couple of [git tutorials](https://ardupilot.org/dev/docs/submitting-patches-back-to-master.html) in our Wiki

Please make sure to read through your PR before force pushing (or at least immediately afterwards :-) )